icons/replit/replit-original.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><path d="M2 1.5A1.5 1.5 0 0 1 3.5 0h7A1.5 1.5 0 0 1 12 1.5V8H3.5A1.5 1.5 0 0 1 2 6.5ZM12 8h8.5A1.5 1.5 0 0 1 22 9.5v5a1.5 1.5 0 0 1-1.5 1.5H12ZM2 17.5A1.5 1.5 0 0 1 3.5 16H12v6.5a1.5 1.5 0 0 1-1.5 1.5h-7A1.5 1.5 0 0 1 2 22.5Z"/></svg>
```

Comment:
This icon needs to be scaled and centered so it fills the entire viewbox

Reply:
> This icon needs to be scaled and centered so it fills the entire viewbox

Didn't saw that. Will update it right now.